wasi-tools/regenerate_bindings.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+
+rm -rf src/wit_world
```

Comment:
This generates the code in src/wit_world and needs to be rerun when the code in the wasi-tools/wit directory changes